storage/src/lsm_tree/object_store_lsm_tree.rs

```diff
@@ -162,8 +157,7 @@ impl ObjectStoreLsmTreeCore {
             let guard = self.rwlock.write();
             // println!("rotate memtable");
             let imm = self.memtable.borrow().clone();
-            *self.memtable.borrow_mut() =
-                Memtable::new(self.options.meta_cache_capacity.0 as usize);
```

Comment:
The performance of `test_concurrent_put` for `ObjectStoreLsmTree` dropped because this wrong line. `meta_cache_capacity` was `1MB` `write_buffer_capacity` was only `64KB`.